cmd/resources/resource_cmds.tmpl

```diff
@@ -0,0 +1,43 @@
+// This file is generated by gen_resources.go; DO NOT EDIT.
+
+package resources
+
+import (
+	"github.com/spf13/cobra"
+
+	"ldcli/internal/analytics"
+	"ldcli/internal/resources"
+)
+
+func AddAllResourceCmds(rootCmd *cobra.Command, client resources.Client, analyticsTracker analytics.Tracker) {
+	// Resource commands{{ range $resName, $resData := .Resources }}
+	gen_{{ $resName }}ResourceCmd := NewResourceCmd(
+        rootCmd,
+        analyticsTracker,
+        "{{ $resData.Name }}",
+        "Make requests (list, create, etc.) on {{ $resData.DisplayName }}",
+        {{ $resData.Description }},
+    )
+    {{ end }}
+
+    // Operation commands{{ range $resName, $resData := .Resources }}{{ range $opName, $opData := $resData.Operations }}
```

Comment:
Could you break these range calls out into separate lines to more easily see the nesting?

Reply:
yeah it'll add an extra line between then comment and the following commands which bothers me only very slightly 🙃